pages/linux/xclip.md

```diff
@@ -0,0 +1,11 @@
+# xclip
+
+> Copy STDIN to clipboard or print clipboard to STDOUT.
+
+- Copy output to clipboard:
+
+`echo 123 | xclip -i`
+
+- Paste clipboard:
```

Comment:
This doesn't exactly paste the clipboard but rather output it, and then you can save it somewhere. So maybe the text should read `Print clipboard's content` ? Just a thought, ultimately you know what this command does best than I do (I'm more of an `pboard` user)